bids/grabbids/bids_layout.py

```diff
@@ -46,10 +46,27 @@ def get_metadata(self, path, **kwargs):
 
         return merged_param_dict
 
-    def get_fieldmap(self, path):
+    def get_fieldmap(self, path, all=False):
+        fieldmaps = self._get_fieldmaps(path)
+
+        if all:
+            return fieldmaps
+        else:
+            if len(fieldmaps) == 1:
+                return fieldmaps[0]
+            elif len(fieldmaps) > 1:
+                raise ValueError("More than one fieldmap found, but 'all' "
+                                 "argument was set to False. Either ensure that "
+                                 "there is only one fieldmap for this image, "
+                                 "or set the 'all' argument to True and handle "
+                                 "the result as a list.")
+            else: # len == 0
+                return fieldmaps
```

Comment:
Won't this return an empty list? Not sure that's what the user is expecting... I know it'll evaluate to `False` in a conditional, but maybe return `None` to remove ambiguity?